inoculate/src/qemu.rs

```diff
@@ -101,8 +101,48 @@ impl Cmd {
     }
 
     #[tracing::instrument(skip(self))]
-    pub fn run_qemu(&self, image: &Path) -> Result<()> {
-        // TODO(eliza): should we `which qemu` here?
+    fn spawn_qemu(&self, qemu: &mut Command, binary: &Path) -> Result<Child> {
+        let (Cmd::Run { qemu_settings, .. } | Cmd::Test { qemu_settings, .. }) = self;
```

Comment:
if we're running tests, we should probably add the `--no-shutdown` QEMU flag, so that it doesn't abruptly terminate and make gdb crash, right? (run mode adds that flag automagically)